### PR TITLE
Feature/les 251 add plyr to project

### PR DESCRIPTION
**Pull request by valer-sky**
https://jr.it-academy.by/browse/LES-251

### CODE REVIEW

**Review by Evgeniy121212 — APPROVED** [review]
Approving ahead, but please, remove "package-lock.json" from the PR

**Review by Terapool — APPROVED** [review]

File: src/components/body/content/videoplayer/Videoplayer.tsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  },
+};
+
+const optionsVideoplayer = {
+  quality: {
+    default: 576,
+    // The options to display in the UI, if available for the source media
+    options: [1080, 720, 576, 480, 360, 240],
+    forced: true,
+
```
Comment (Kurilo-Pavel) [inline src/components/body/content/videoplayer/Videoplayer.tsx:64]: Please delete empty line 64

File: src/components/body/content/videoplayer/Videoplayer.tsx [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+  return (
+    <VideoPlayer />
+  );
+}
+
+export const VideoPlayer: React.FC = () => {
+  return (
+    <div className ="player">
+      <Plyr options={optionsVideoplayer}
+        source={videoSrc}
```
Comment (Kurilo-Pavel) [inline src/components/body/content/videoplayer/Videoplayer.tsx:116]: Please add Tab in line 116

File: src/components/body/content/videoplayer/index.scss [hunk 3]
```diff
@@ -0,0 +1,11 @@
+.player {
+    --plyr-captions-background: rgba(0, 0, 0, 0.8); /*The color for the background of captions.*/
+    --plyr-captions-text-color: white; /* The color used for the captions text.*/
+  }
+.wrapper {
```
Comment (Kurilo-Pavel) [inline src/components/body/content/videoplayer/index.scss:5]: Please add empty line between lenes 4 and 5

**Review by OxygenO22 — APPROVED** [review]

**Pull request merged**